### PR TITLE
Remove chatml validation from sdk

### PR DESCRIPTION
**Pull request by HowieG**
## 📥 Pull Request

**📘 Description**
Moving chatml coercion to api server so removing from here

### CODE REVIEW

**Comment by HowieG** [comment]
Don't merge yet. Need to update api

File: agentops/client.py [hunk 1]
```diff
@@ -32,53 +31,48 @@ class Client(metaclass=MetaClient):
...
-            parent_key = environ.get('AGENTOPS_PARENT_KEY', None)
 
-        if not endpoint:
-            endpoint = environ.get('AGENTOPS_API_ENDPOINT', 'https://api.agentops.ai')
-
-        self.config = Configuration(api_key,
-                                    parent_key,
-                                    endpoint,
-                                    max_wait_time,
-                                    max_queue_size)
```
Comment (braelyn-ai) [inline agentops/client.py]: we should raise the error again or at least log it. if we only return the dev will never know something went wrong with AO

File: agentops/langchain_callback_handler.py [hunk 2]
```diff
@@ -433,6 +433,7 @@ async def on_llm_end(
             # TODO: more descriptive error
             error_event = ErrorEvent(trigger_event=self.events.llm[str(run_id)],
                                      details="on_llm_end: No generations", timestamp=get_ISO_time())
+                                     details="on_llm_end: No generations", timestamp=get_ISO_time())
```
Comment (siyangqiu) [inline agentops/langchain_callback_handler.py:436]: dupe

File: agentops/event.py [hunk 3]
```diff
@@ -92,6 +92,7 @@ class LLMEvent(Event):
     completion: str | object = None
     completion_tokens: Optional[int] = None
     model: Optional[Models | str] = None
+    model: Optional[Models | str] = None
```
Comment (siyangqiu) [inline agentops/event.py]: dupe

File: agentops/llm_tracker.py [hunk 4]
```diff
@@ -121,6 +121,7 @@ def handle_stream_chunk(chunk: ChatCompletionChunk):
             )
 
             try:
+                # NOTE: prompt/completion usage not returned in response when streaming
```
Comment (siyangqiu) [inline agentops/llm_tracker.py]: dupe

File: agentops/llm_tracker.py [hunk 5]
```diff
@@ -181,7 +182,7 @@ async def async_generator():
             self.llm_event.agent_id = check_call_stack_for_agent_id()
             self.llm_event.prompt = kwargs["messages"]
             self.llm_event.prompt_tokens = response.usage.prompt_tokens
-            self.llm_event.completion = response.choices[0].message.model_dump().get('content')
```
Comment (siyangqiu) [inline agentops/llm_tracker.py:184]: make sure this isn't a merge conflict

**Review by siyangqiu — APPROVED** [review]
Some dupes, but otherwise good

File: agentops/llm_tracker.py (hunk 5 above)
Comment (HowieG) [reply agentops/llm_tracker.py:184]: intentional

**Pull request merged**